app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -196,16 +204,20 @@ public Result getUploads(String startTimeString, String endTimeString) {
      * @param endTimeString
      * @return
      */
-    public Result getUploadsForStudy(String studyId, String startTimeString, String endTimeString) throws EntityNotFoundException {
+    public Result getUploadsForStudy(String studyIdString, String startTimeString, String endTimeString) throws EntityNotFoundException {
+        if (StringUtils.isBlank(studyIdString)) {
+            throw new BadRequestException("Invalid study Id.");
+        }
```

Comment:
Again, input validation and business logic should be in the service, not in the controller.